src/components/Locations.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import axios from "axios";
+
+class Locations extends Component {
+    constructor() {
+        super();
+        this.state = {
+            locations: [],
+            displayList: false,
+        }
+    }
+
+    grabLocations = async () => {
+        const { data } = await axios.get("https://ghibliapi.herokuapp.com/locations/");
```

Comment:
Nice deconstruction of the return from your `await` function call!

Reply:
Thank you